test/UnsetOption.lean

```diff
@@ -0,0 +1,18 @@
+import Mathlib.Tactic.UnsetOption
+import Mathlib.Tactic.RunCmd
+
+set_option pp.all true
+
+example : True := by
+  run_tac
+    let t : Option Bool := (← Lean.MonadOptions.getOptions).get? `pp.all
+    guard (t == true)
+  trivial
+
+unset_option pp.all
+
+example : True := by
```

Comment:
Add a test where `unset_option` is used in a section, to ensure that the value is un-unset once the section is closed